test/test_jit.py

```diff
@@ -10551,6 +10551,76 @@ def addmm_grad_test(b, x, w):
         self.assertEqual(w.grad, w_ref.grad)
         self.assertEqual(b.grad, b_ref.grad)
 
+    def test_layer_norm_grad(self):
+        class MyLayerNorm(torch.nn.Module):
+            __constants__ = ['norm_shape']
+
+            def __init__(self, norm_shape):
+                super(MyLayerNorm, self).__init__()
+                self.norm_shape = norm_shape
+
+            def forward(self, x: torch.Tensor, w: Optional[torch.Tensor], b: Optional[torch.Tensor]):
+                o = torch.nn.functional.relu(x)
+                o = torch.nn.functional.layer_norm(o, self.norm_shape, w, b)
+                return o
+
+        # Initialize param and input values
+        x_init = torch.rand(3, 2)
+        norm_shape = [2]
+        w_init = torch.rand(norm_shape)
+        b_init = torch.rand(norm_shape)
+
+        layer_norm = torch.jit.script(MyLayerNorm(norm_shape))
+
+        scenarios = [[False, False], [True, False], [True, True]]
```

Comment:
Let's put a comment that `[False, True]` is not supported by native_layer_norm_backward implementation. So at least we know it's not an oversight on our end.